src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,39 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        Manufacturer manufacturer = new Manufacturer(3L,"Audi","German");
+        List<Driver> drivers = new ArrayList<>();
+        Car car = new Car("audi",manufacturer,drivers);
+
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        carService.create(car);
+
+        Car car1 = carService.get(3L);
```

Comment:
if I download your app if this will valid data for me. I mean that will be I have car with id 3?
оскільки id це auto-increment значення то ми не можемо знати, що на іншій машині буде згенеровано те саме значення id і при запуску виникне помилка. Тому звертатися на пряму по id не найкраща ідея, краще передавати щось типу carService.get(car.getId())

and I think it is not good naming